### ISSUE TITLE
integer and float configs are not passed in laconia context

### ISSUE THREAD

**Issue by reestolonio** [laconiajs/laconia#637]
**Describe the bug**
Float and Integer configs are not included in laconia context.

**To Reproduce**
In lambda environment, I defined `LACONIA_CONFIG_KEY_EXPIRATION: integer:10000`,

```
const instance = ({ keyExpiration }) => ({
    myService: new MyService(keyExpiration)
})

const _handler = laconia(apig(app))
  .register(envVarInstances())
  .register(instance)
```

**Expected behavior**
`keyExpiration` should be available in laconia context

**Actual behavior**
`keyExpiration` is not in laconia context

**Additional context**
`IntegerConfigConverter` and `FloatConfigConverter` are not registered in `EnvVarConfigFactory`

**Comment by reestolonio** [comment]
I can submit a PR for this.

**Comment by reestolonio** [comment]
PR is in #638

**Comment by geoffdutton** [comment]
Closed in #638

**Comment by ceilfors** [comment]
@all-contributors please add @reestolonio for bug
